src/visit.rs

```diff
@@ -109,3 +116,29 @@ fn visit_directories(
 
     Ok(())
 }
+
+fn normalize_import(
+    import: &str,
+    root: &Path,
+    current: &Path,
+) -> Result<PathBuf, StripPrefixError> {
+    let full_path = current.join(Path::new(&import));
+
+    let file_name = full_path.file_name().unwrap_or_default();
+    let directory_path = full_path
+        .parent()
+        .expect("Expected a parent directory to exist");
+
+    match canonicalize(directory_path) {
+        Ok(canonicalized_directory_path) => {
```

Comment:
nit: because you're not actually using the `Err` value at all you can simplify this

```suggestion
    if let Ok(canonicalized_directory_path) = canonicalize(directory_path) {
```